### DISCUSSION TITLE
lesson 6: i having a issue in <path>:<contractname>

### DISCUSSION THREAD

**Discussion by Asyedabdulrahman** [Cyfrin/foundry-full-course-cu#905] [General]
i am having an issue in while deploying contract. 

here is my error :-
rahman33@RAHMAN:~/foundry-simple-storage-f23$ forge script script/deploysimplestorage.s.sol --rpc-url http://127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80

Error: 
contract source info format must be `<path>:<contractname>` or `<contractname>`
 can anyone give solution for this error

**Comment by theirrationalone** [comment]
You're not targeting script contract correctly, is indicated by logs. 

```bash
forge script script/deploysimplestorage.s.sol --rpc-url http://127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 ❌

forge script script/deploysimplestorage.s.sol:DeploySimpleStorage --rpc-url http://127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 ✅
```

That `DeploySimoleStorage` is the contract name resides inside `deploysimplestorage.s.sol` file. 

;`)

**Reply by essnuked** [reply]
so what can we do to fix it? cause i have same issue

**Reply by EngrPips** [reply]
Did have your deployment command like the below?

```js
forge script script/deploysimplestorage.s.sol:DeploySimpleStorage --rpc-url http://127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 ✅
```
